overlord/snapstate/snapstate.go

```diff
@@ -302,8 +306,68 @@ func Remove(s *state.State, name string) (*state.TaskSet, error) {
 
 // Rollback returns a set of tasks for rolling back a snap.
 // Note that the state must be locked by the caller.
-func Rollback(s *state.State, snap, ver string) (*state.TaskSet, error) {
-	return nil, fmt.Errorf("rollback not implemented")
+func Rollback(s *state.State, name, ver string) (*state.TaskSet, error) {
```

Comment:
Yes, it is reset in https://github.com/snapcore/snapd/pull/1275/files#diff-82a44be6f243493a55d2ee2fe6f0addfR660. I added code that will prevent two rollbacks now.